src/team-content.js

```diff
@@ -1,4 +1,18 @@
 const teamContent = [
+  {
+    name: 'Zain Khoja',
+    title: 'Systems Designer Intern',
+    handle: 'zainkho',
+    cssFeature: 'fill: currentColor;',
+    questionOne: `I love organization and lists. I have a lists for things from movies to boba tea orders. I like to bring this into my design work as well, creating components and cohesiveness across platforms to make it easier for people to create great products.`,
+    questionTwo: `When I first began learning about design, I was inspired by the people who contributed back to the community by teaching and opening up to students. Resources like [Students Who Design](https://studentswho.design/) and [Design Details](https://designdetails.fm) helped me figure out how I wanted to make a positive impact in tech.
+
+  More recently, I've been inspired by design agencies and companies that are testing the boundaries of design outside of what is accepted as 'common pratice'. I enjoy seeing unique animation and motion used to convey information, emotion, and storytelling.`,
+    favoriteTools: 'Sketch, Figma, Atom, Dropbox Paper, Twitter, Ping Pong, Ukulele',
+    avatar: 'https://avatars1.githubusercontent.com/u/6846673?v=4',
```

Comment:
Looks like this avatar doesn't have the "primer" filter. Can you swap it with the one that @ashygee sent you?